src/modules/freia/configs/amor.json

```diff
@@ -3,30 +3,48 @@
   "Detector" : "Freia",
   "InstrumentGeometry" : "AMOR",
 
+  "Version" : 1,
+  "Comment" : "v1: strip and wire thresholds per cassette, not per channel",
+  "Date" : "2023 01 01",
+  "Info" : "Generated by slowcontrols.exe version x.y, git commit hash",
+
   "History" : [
     "2023-09-12 Change hybrid order",
-    "2023-09-25 Added hybrid id's"
+    "2023-09-25 Added hybrid id's",
+    "2023-10-06 add info fields to config file"
   ],
 
-  "Version" : "n/a",
-
   "WireChOffset" : 16,
 
   "Config" : [
-    { "CassetteNumber":  0, "Ring" :  2, "FEN": 0, "Hybrid" :  3, "HybridId" : "a08000e4fccca0803410082006704410"},
-    { "CassetteNumber":  1, "Ring" :  2, "FEN": 0, "Hybrid" :  2, "HybridId" : "a08000753877a0803410082006704410"},
-    { "CassetteNumber":  2, "Ring" :  2, "FEN": 0, "Hybrid" :  1, "HybridId" : "a0800084049ca0803410082106704410"},
-    { "CassetteNumber":  3, "Ring" :  2, "FEN": 0, "Hybrid" :  0, "HybridId" : "a08000e3b432a0803410082006704410"},
-    { "CassetteNumber":  4, "Ring" :  1, "FEN": 0, "Hybrid" :  4, "HybridId" : "a080003b54b4a0803410082006704410"},
-    { "CassetteNumber":  5, "Ring" :  1, "FEN": 0, "Hybrid" :  3, "HybridId" : "a08000e0103ea0803410082106704410"},
-    { "CassetteNumber":  6, "Ring" :  1, "FEN": 0, "Hybrid" :  2, "HybridId" : "a0800004349aa0803410082006704410"},
-    { "CassetteNumber":  7, "Ring" :  1, "FEN": 0, "Hybrid" :  1, "HybridId" : "a0800075c091a0803410082006704410"},
-    { "CassetteNumber":  8, "Ring" :  1, "FEN": 0, "Hybrid" :  0, "HybridId" : "a08000d1448ba0803410082006704410"},
-    { "CassetteNumber":  9, "Ring" :  0, "FEN": 0, "Hybrid" :  4, "HybridId" : "a0800073b01ea0803410082006704410"},
-    { "CassetteNumber": 10, "Ring" :  0, "FEN": 0, "Hybrid" :  3, "HybridId" : "a08000a3385da0803410082006704410"},
-    { "CassetteNumber": 11, "Ring" :  0, "FEN": 0, "Hybrid" :  2, "HybridId" : "a08000f3bcb9a0803410082006704410"},
-    { "CassetteNumber": 12, "Ring" :  0, "FEN": 0, "Hybrid" :  1, "HybridId" : "a0800040c8cfa0803410082006704410"},
-    { "CassetteNumber": 13, "Ring" :  0, "FEN": 0, "Hybrid" :  0, "HybridId" : "a0800006b882a0803410082006704410"}
+    { "CassetteNumber":  0, "Ring" :  2, "FEN": 0, "Hybrid" :  3, "Thresholds" : [[42],[43]],
+      "HybridId" : "a08000e4fccca0803410082006704410"},
```

Comment:
I suggest to  move the HybridID in the same line, because it's much more readable in that way. This applies all this config block.